src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,100 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { actions } from '../../features/currentTodo';
+import { Todo } from '../../types/Todo';
 
 export const TodoModal: React.FC = () => {
+  const [user, setUser] = useState<User>();
+  const [todo, setTodo] = useState<Todo | null>();
+  const [loading, setLoading] = useState(true);
+
+  const selectedTodo = useAppSelector(store => store.currentTodo);
+  const dispatch = useAppDispatch();
+
+  const getUserFromServer = async () => {
+    let selectedUser;
```

Comment:
Are you sure you need a `let` here? Do you need to setUser if you don't have a correct response from the getUser()? Maybe use try/catch here?